pkg/utils/utils.go

```diff
@@ -2,6 +2,7 @@ package utils
 
 import (
 	"bufio"
+	"math/rand"
```

Comment:
just in case, math/rand is not secure comparing to crypto/rand

Reply:
good point